samples/snippets/csharp/VS_Snippets_Wpf/WindowClosingSnippets/CSharp/DataWindow.xaml.cs

```diff
@@ -1,4 +1,3 @@
-//<SnippetWindowClosingCODEBEHIND1>
 using System; // EventArgs
 using System.ComponentModel; // CancelEventArgs
 using System.Windows; // window
```

Comment:
@mairaw Here's a PR removing the comments: https://github.com/dotnet/docs/pull/2060.